pkg/tsdb/elasticsearch/response_parser.go

```diff
@@ -118,8 +119,9 @@ func parseResponse(ctx context.Context, responses []*es.SearchResponse, targets
 
 			result.Responses[target.RefID] = queryRes
 		}
-		resSpan.End()
+		instrumentation.UpdatePluginParsingResponseDurationSeconds(ctx, time.Since(start), "ok")
```

Comment:
I think we should also do a 

```
instrumentation.UpdatePluginParsingResponseDurationSeconds(ctx, time.Since(start), "error")
```

after line 114. WDYT?

Reply:
Good catch! Added in https://github.com/grafana/grafana/pull/75628/commits/397b5f36c9b9fb458a4fcfc002c839335c0cc0cf + I have also added `es.StageParseResponse` to logging that was missing.